schemas/common_definitions.json

```diff
@@ -63,11 +66,29 @@
           "items": {
             "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
           }
+        },
+        "definition": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/definition"
         }
       },
       "additionalProperties": false
     }
   },
+  "definition": {
+    "description": "Definition title and contents.",
+    "items": {
+      "type": "object",
+      "required": ["title", "contents"],
+      "properties": {
+        "title": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+        },
+        "contents": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+        }
+      }
+    }
+  },
```

Comment:
Question definitions should be updated to point to this partial as the object is identical.